### PR TITLE
[BULK] - DocuTune - Rebranding of Azure Active Directory to Microsoft Entra terminology (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: azps-0.10.0/Az.Compute/Set-AzVMDiskEncryptionExtension.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ It is advised that you save your work on the virtual machine before you run this
 
 ## EXAMPLES
 
-### Example 1: Enable encryption using Azure AD Client ID and Client Secret
+<a name='example-1-enable-encryption-using-azure-ad-client-id-and-client-secret'></a>
```
Comment (mikefrobbins) [inline azps-0.10.0/Az.Compute/Set-AzVMDiskEncryptionExtension.md:49]: This line is causing the build process to generate an error.

File: azps-0.10.0/Az.Compute/Set-AzVMDiskEncryptionExtension.md [hunk 2]
```diff
@@ -59,9 +61,11 @@ $KeyVaultResourceId = $KeyVault.ResourceId
 Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId
 ```
 
-This example enables encryption using Azure AD client ID, and client secret.
+This example enables encryption using Microsoft Entra client ID, and client secret.
+
+<a name='example-2-enable-encryption-using-azure-ad-client-id-and-client-certification-thumbprint'></a>
```
Comment (mikefrobbins) [inline azps-0.10.0/Az.Compute/Set-AzVMDiskEncryptionExtension.md:66]: This line is causing the build process to generate an error.

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**